pxb/jenkins/pxb-package-testing.groovy

```diff
@@ -0,0 +1,148 @@
+library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+]) _
+
+product_action_playbooks = [
+    pxb80: [
+        install: 'pxb_80.yml',
+        upgrade: 'pxb_80_upgrade.yml',
+        upstream: 'pxb_80_upstream.yml',
+        tarball: 'pxb_80_tarball.yml'
+    ],
+    pxb24: [
+        install: 'pxb_24.yml',
+        upgrade: 'pxb_24_upgrade.yml',
+        upstream: 'pxb_24_upstream.yml',
+        tarball: 'pxb_24_tarball.yml'
+    ]
+]
+
+void runPlaybook(String action_to_test) {
+    def playbook = product_action_playbooks[params.product_to_test][action_to_test]
+    def playbook_path = "package-testing/playbooks/${playbook}"
+
+    sh '''
+        git clone --depth 1 "${git_repo}"
+
+        if [ -f /usr/bin/yum ]; then
+            sudo yum -y update
+            sudo yum -y install epel-release
+            sudo yum -y install ansible
+        fi
+
+        if [ -f /usr/bin/apt-get ]; then
+            sudo DEBIAN_FRONTEND=noninteractive apt-get update
+            sudo DEBIAN_FRONTEND=noninteractive apt-get -y install python
+            curl https://bootstrap.pypa.io/pip/2.7/get-pip.py -o get-pip.py
+            sudo python get-pip.py
+            sudo python -m pip install ansible
```

Comment:
why we cannot install ansible from repo?

Reply:
This is being ported over as-is from an existing job, so I don't really know the reasoning behind this decision. My guess is that installing it from pip allows us to get a more up-to-date version, and more consistent across distributions, than the distributions' repositories.

Reply:
I suggest not just move them as-is but improve and refactor. 
Using pip is not a common way as otherwise we can say ` hey why do we need percona packages? we have github let's compile it!`
please investigate the possibility of installing ansible from rerpos. 
Also as I remember for ps-testing jobs you installed ansible from repos, weren't you?

Reply:
Done! Same solution as the package-testing-ps jobs, meaning, Stretch installs from the Trusty PPA since the version on repos is too old, every other distro installs from repo as those are up-to-date enough. Successful builds at https://pxb.cd.percona.com/job/pxb-package-testing-all-jenkins-test/4/.